### PR TITLE
Add serial English auction minter

### PR DESCRIPTION
**Pull request by ryley-o**
## Description of the change

Add support for indexing a new serial English Auction (SEA) minter.

This PR includes all updates and tests associated with indexing the new minter added in: https://github.com/ArtBlocks/artblocks-contracts/pull/504

No schema changes are required. This is due to using existing fields and generic json fields to store all new minter parameters.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203665500354291

### CODE REVIEW

File: src/minter-suite-mapping.ts
Comment (jakerockland) [inline src/minter-suite-mapping.ts:514]: In this case here, and below, should we log a warning within these guards?

If we do that, should we add a utility function?

File: abis/IFilteredMinterSEAV0.json
Comment (jakerockland) [inline abis/IFilteredMinterSEAV0.json:1]: Should we update this repo to import the ABI npm package so that we don't need to manually import like this?

**Review by jakerockland — APPROVED** [review]

File: abis/IFilteredMinterSEAV0.json
Comment (ryley-o) [reply abis/IFilteredMinterSEAV0.json:1]: This is a good callout. I would like to do this in a follow-on PR to isolate the change, and have added an issue for this here: https://github.com/ArtBlocks/artblocks-subgraph/issues/197

Comment (jakerockland) [reply abis/IFilteredMinterSEAV0.json:1]: SGTM!

Comment (jakerockland) [reply abis/IFilteredMinterSEAV0.json:1]: Thank you!! :)

**Review by yoshiwarab — APPROVED** [review]

File: src/minter-suite-mapping.ts
Comment (ryley-o) [reply src/minter-suite-mapping.ts]: good point - realized we don't even need these null minter guards because `loadOrCreateMinter` never returns null. This was some old habits from when we didn't have the helper function. Updated in 061dd92 🙏 

Note that if we do have guards like these, in general, I would prefer to log a warning to help keep us aware of any ongoing issues in deployed subgraphs!

**Comment by ryley-o** [comment]
Tagging this with `do not merge` until smart contract repo's PR is merged 🙏

**Comment by ryley-o** [comment]
Oops I guess the do not merge tag still merges 😓 this is fine, and any updates from any smart contract iteration will be made in subsequent PRs.

**Pull request merged**
